packages/flutter/test/material/mergeable_material_test.dart

```diff
@@ -55,7 +55,7 @@ BorderRadius getBorderRadius(WidgetTester tester, int index) {
   final List<Element> containers = tester.elementList(find.byType(Container))
                                    .toList();
 
-  final Container container = containers[index + 2].widget;
+  final Container container = containers[index].widget;
```

Comment:
The test depends on the structure of the object hierarchy built by the Material widget.  Material now generates a PhysicalModel instead of two nested Containers, so the test code that skips over the containers needed to be adjusted.